src/components/ebay-icon/symbols/chevron-left-small/index.marko

```diff
@@ -1,29 +1,5 @@
-<% if (typeof window !== 'undefined') console.error('ds4 icon not found: chevron-left-small') %>
-<symbol id="icon-chevron-left-small" viewBox="0 0 16 16">
-  <g fill="#111820" fill-rule="nonzero">
-    <g transform="translate(5.5 4.5)">
-      <circle cx="2.3" cy="6.1" r="1" />
-      <path
-        d="M1.8 4.7c0-.6 0-1.1.3-1.4.4-.4.9-.5 1.1-.6.3-.2.6-.5.6-.8 0-.3-.2-.8-1.3-.8-1 0-1.4.6-1.4 1.2H.3c0-1.3.8-2 2.2-2C4.6.4 4.7 1.7 4.7 2c0 .4-.2.8-.7 1.1-.5.4-.9.4-1.1.7-.2.2-.2.5-.2 1h-.9z"
-      />
-    </g>
-    <g transform="translate(1 1)">
-      <circle cx="7" cy=".7" r="1" />
-      <circle cx="10.2" cy=".7" r="1" />
-      <circle cx="13.3" cy=".7" r="1" />
-      <circle cx=".7" cy=".7" r="1" />
-      <circle cx="3.8" cy=".7" r="1" transform="rotate(90 3.8 .7)" />
-      <circle cx="7" cy="13.3" r="1" transform="rotate(180 7 13.3)" />
-      <circle cx="3.8" cy="13.3" r="1" transform="rotate(180 3.8 13.3)" />
-      <circle cx=".7" cy="13.3" r="1" transform="rotate(180 .7 13.3)" />
-      <circle cx="13.3" cy="13.3" r="1" transform="rotate(180 13.4 13.3)" />
-      <circle cx="10.2" cy="13.3" r="1" transform="rotate(-90 10.2 13.3)" />
-      <circle cx="13.3" cy="7" r="1" transform="rotate(90 13.3 7)" />
-      <circle cx="13.3" cy="10.2" r="1" transform="rotate(90 13.3 10.2)" />
-      <circle cx="13.3" cy="3.8" r="1" transform="rotate(180 13.3 3.8)" />
-      <circle cx=".7" cy="7" r="1" transform="rotate(-90 .7 7)" />
-      <circle cx=".7" cy="3.8" r="1" transform="rotate(-90 .7 3.8)" />
-      <circle cx=".7" cy="10.2" r="1" />
-    </g>
-  </g>
+<symbol id="icon-chevron-left-small" viewBox="0 0 14 24">
+  <path
+    d="M13.993 2.45L2.449 13.992 0 11.544 11.545 0l2.449 2.45h-.001zM2.449 9.096l11.545 11.545-2.449 2.45L0 11.546l2.449-2.45z"
```

Comment:
Did we lose the fill?

Reply:
I copied the `chevron-left` code to the small one. So there is no fill there either. 
The icon there initially was the `?` missing icon one. 
Also this is only for the DS4 which shouldn't have a fill I believe. 

I can rerun the generateSVG script, but there were a lot of changes. Maybe we can issue another PR for that?

Reply:
Ah ... that makes sense. Thanks.

Reply:
Yes, lets wait on rerunning the `generateSVG` script.